OptimizelySDK/Config/PollingProjectConfigManager.cs

```diff
@@ -34,33 +34,29 @@ public abstract class PollingProjectConfigManager : ProjectConfigManager
         private TimeSpan PollingInterval;
         public bool IsStarted { get; private set; }
         private bool scheduleWhenFinished = false;
+        public bool AutoUpdate { get; private set; }
 
         private ProjectConfig CurrentProjectConfig;
         private Timer SchedulerService;
-
         protected ILogger Logger { get; set; }
         protected IErrorHandler ErrorHandler { get; set; }
         protected TimeSpan BlockingTimeout;
         protected TaskCompletionSource<bool> CompletableConfigManager = new TaskCompletionSource<bool>();
         // Variables to control blocking/syncing.
-        public object mutex = new Object();
+        public int resourceInUse = 0;
 
-        protected event Action ProjectConfig_Notification;
+        public event Action NotifyOnProjectConfigUpdate;
 
-        public PollingProjectConfigManager(TimeSpan period, TimeSpan blockingTimeout, ILogger logger = null, IErrorHandler errorHandler = null, bool StartByDefault = true)
+        public PollingProjectConfigManager(TimeSpan period, TimeSpan blockingTimeout, bool autoUpdate = false, ILogger logger = null, IErrorHandler errorHandler = null)
```

Comment:
`autoUpdate` defaults to `true` in server-side JS, and I believe Java also.

Reply:
Sure, I will make this change and update it.